x-pack/plugins/canvas/canvas_plugin_src/functions/common/font.js

```diff
@@ -5,55 +5,74 @@
  */
 
 import inlineStyle from 'inline-style';
+import { i18n } from '@kbn/i18n';
 import { openSans } from '../../../common/lib/fonts';
 
 export const font = () => ({
   name: 'font',
   aliases: [],
   type: 'style',
-  help: 'Create a font style',
+  help: i18n.translate('xpack.canvas.functions.fontHelpText', {
+    defaultMessage: 'Create a font style',
+  }),
   context: {
     types: ['null'],
   },
   args: {
     size: {
       types: ['number'],
-      help: 'Font size (px)',
+      help: i18n.translate('xpack.canvas.functions.font.argsSizeHelpText', {
+        defaultMessage: 'Font size (px)',
+      }),
       default: 14,
     },
     lHeight: {
       types: ['number'],
       aliases: ['lineHeight'],
-      help: 'Line height (px)',
+      help: i18n.translate('xpack.canvas.functions.font.argsLHeightHelpText', {
+        defaultMessage: 'Line height (px)',
+      }),
     },
     family: {
       types: ['string'],
       default: `"${openSans.value}"`,
-      help: 'An acceptable CSS web font string',
+      help: i18n.translate('xpack.canvas.functions.font.argsFamilyHelpText', {
+        defaultMessage: 'An acceptable CSS web font string',
+      }),
     },
     color: {
       types: ['string', 'null'],
-      help: 'Text color',
+      help: i18n.translate('xpack.canvas.functions.font.argsColorHelpText', {
+        defaultMessage: 'Text color',
+      }),
     },
     weight: {
       types: ['string'],
-      help:
-        'Set the font weight, e.g. normal, bold, bolder, lighter, 100, 200, 300, 400, 500, 600, 700, 800, 900',
+      help: i18n.translate('xpack.canvas.functions.font.argsWeightHelpText', {
+        defaultMessage:
+          'Set the font weight, e.g. normal, bold, bolder, lighter, 100, 200, 300, 400, 500, 600, 700, 800, 900',
```

Comment:
put normal, bold, bolder, lighter, 100, 200, 300, 400, 500, 600, 700, 800, 900 to values like fontWeightValuesList